docs/modules/user-guides/pages/appium-setup.adoc

```diff
@@ -0,0 +1,427 @@
+selenium.grid.capabilities.app=/Users/myDirectory/application.apk
+----
+
+==== Real devices properties
+
+While Android devices require only one additional property, for iOS you need to set three more.
+
+IMPORTANT: See the relevant sections for defining these properties for xref:appium-setup.adoc#_define_properties_for_running_a_real_apple_device[iOS] and xref:appium-setup.adoc#_android_device_udid[Android] 
+
+== Appium Inspector (Optional)
+
+https://github.com/appium/appium-inspector[Appium Inspector] is basically just an Appium client with a user interface. It allows to interact with application elements and Appium commands in a user friendly way.
+You can get the latest version of Inspector application following the https://github.com/appium/appium-inspector/releases[link].
+
+.Click to view capabilities set up in Inspector
+[%collapsible]
+====
+image::appium-inspector-capabilities-android.png[]    
+image::appium-inspector-capabilities-ios.png[]
+====
+
+== Running tests on Simulators
+
+VIVIDUS Sample Tests is a https://github.com/vividus-framework/vividus-sample-tests[GitHub repository]. Its puropose is to demo VIVIDUS capabilities via sample tests. 
+
+Let's use it as an example for running mobile tests via Appium.
+
+. *Assumptions*
++
+You have installed and properly set up the following:
+
+* [*] IDE
+* [*] Appium Server
+* [*] AVD Manager (Android)
+* [*] Simulator application (iOS)
+
+. *Cloning and importing the repository*
++
+Clone vividus-sample-tests project using the following command:
++
+[source,shell]
+----
+git clone --recursive git@github.com:vividus-framework/vividus-sample-tests.git
+----
++
+Then use xref:ROOT:getting-started.adoc#_create_tests_project[this detailed guide] to proceed with importing and building the project.
+
+. *Overriding properties*
++
+Since we are only interested in the mobile application story, add the following properties to the overriding.properties file. Choose one of the platforms and make sure properties match with your simulator set up.
++
+.Click to expand overriding.properties
+[%collapsible]
+====
+[source,property]
+----
+# Update properties with your device type and comment unnecessary block.
+
+# Minimum iOS setup
+selenium.grid.capabilities.appiumVersion=1.20.2
+selenium.grid.url=http://127.0.0.1:4723/wd/hub
+configuration.profiles=mobile_app/ios
+configuration.suites=mobile_app/ios
+configuration.environments=
+selenium.grid.capabilities.deviceName=iPad Pro (9.7-inch)
+selenium.grid.capabilities.platformName=iOS
+selenium.grid.capabilities.automationName=XCUITest
+
+# Minimum Android setup
+selenium.grid.capabilities.appiumVersion=1.20.2
+selenium.grid.url=http://127.0.0.1:4723/wd/hub
+configuration.profiles=mobile_app/android
+configuration.suites=mobile_app/android
+configuration.environments=
+selenium.grid.capabilities.deviceName=Lenovo
+selenium.grid.capabilities.platformName=android
+selenium.grid.capabilities.automationName=UIAutomator2
+----
+====
```

Comment:
https://github.com/vividus-framework/vividus/pull/2925/files#r917651417 same here, so end users can copy the content into clipboard without the need to remote unnecessary stuff